### PR TITLE
Log Cron to Lambda wrapper

### PR DESCRIPTION
**Pull request by fcamilleri22**
Just a blank config file to contain the url of mxdev and a script to pipe over log output to said url. Simple.

### CODE REVIEW

File: launchCronAndLogToMXDev.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+MXDEV_URL=$(cat ~/MXControl/config/MXDEV_URL)
```
Comment (edobry) [inline launchCronAndLogToMXDev.sh:2]: load in the profile rather than doing this, tbh

File: launchCronAndLogToMXDev.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+function postToLambda {
+    message="$(echo $1 | sed "s/^.*]: /**AUTO: **/")"
+    curl --silent -X POST -d "{\"source\": \"MXControl\",\"message\":\"$message\"}" "$MXDEV_URL"
```
Comment (edobry) [inline launchCronAndLogToMXDev.sh]: REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Review by edobry — APPROVED** [review]
fine. BUT I'M NOT HAPPY ABOUT IT.

File: launchCronAndLogToMXDev.sh (hunk 2 above)
Comment (fcamilleri22) [reply launchCronAndLogToMXDev.sh]: what? it's in a function :P

Comment (edobry) [reply launchCronAndLogToMXDev.sh]: the ree is about the JSON >:c

File: launchCronAndLogToMXDev.sh (hunk 1 above)
Comment (fcamilleri22) [reply launchCronAndLogToMXDev.sh:2]: no mxprofile in MXUtil. We could change that, but yeah.

**Pull request merged**